dbt/adapters/redshift/connections.py

```diff
@@ -387,6 +387,7 @@ def exponential_backoff(attempt: int):
             redshift_connector.OperationalError,
             redshift_connector.DatabaseError,
             redshift_connector.DataError,
+            redshift_connector.InterfaceError,
```

Comment:
The issue mentions both `InterfaceError` and `BrokenPipe`. Is the exclusion of `BrokenPipe` intentional?

Reply:
Yea `InterfaceError` wraps `BrokenPipe`: https://github.com/aws/amazon-redshift-python-driver/pull/185/files#diff-477a97c50d560903186137b2ed2a7b538988d88ccfe5178fa6d768df5a308f1eR745